Apps/Playground/0.71/android/app/build.gradle

```diff
@@ -102,6 +102,28 @@ android {
         versionName "1.0"
     }
 
+    packagingOptions {
+        pickFirst 'lib/*/libfabricjni.so'
```

Comment:
Isn't `pickFirst` typically needed when there are duplicate libs? If that's the case, why are there duplicates of all these? Or is something else going on? Maybe add some comments here as well.

Reply:
This is the error I get when building a freshly created playground with rn0.71 & Babylon 
![image](https://github.com/BabylonJS/BabylonReactNative/assets/1312968/476e867b-824c-443b-9456-2525a756c394)

Reply:
too many .so files coming from react-native were copied into assembled and into the npm. jnilibs is now more restrictive on files.